test/probes/tedious.test.js

```diff
@@ -207,9 +215,16 @@ describe('probes.tedious ' + pkg.version, function () {
     }
     const connection = new Connection(settings)
 
-    connection.on('connect', function () {
+    connection.on('connect', function (err) {
+      if(err) {
+        throw err
+      }
       connection.execSql(fn())
+      connection.close()
     })
+
+    // api changed between versions
+    if (semver.gte(pkg.version, '10.0.0')) connection.connect()
```

Comment:
Meaning an explicit `connect()` call is needed for newer version of tedious, right?

Reply:
Yep. Temp stopgap. We'll have to revisit again for 11.x and 12.x (AO-20490) due to other issues.